src/sentry/issues/auto_source_code_config/integration_utils.py

```diff
@@ -0,0 +1,26 @@
+from sentry.constants import ObjectStatus
+from sentry.integrations.base import IntegrationInstallation
+from sentry.integrations.services.integration import integration_service
+from sentry.models.organization import Organization
+
+from .constants import SUPPORTED_INTEGRATIONS
+from .errors import InstallationNotFoundError
+
+
+def get_installation(organization: Organization) -> IntegrationInstallation:
```

Comment:
This is a simplified version of this:
https://github.com/getsentry/sentry/blob/c6c2076e7ef5db14dc3e0be1882cd19a7fa6132e/src/sentry/issues/auto_source_code_config/task.py#L187-L208